gradio/blocks.py

```diff
@@ -2561,6 +2557,10 @@ def reverse(text):
         self.head_paths = head_paths
         self._set_html_css_theme_variables()
 
+        if self._is_running_in_reload_thread:
```

Comment:
Moved this down so that we can set the latest values of theme, etc when they're set via launch